### PR TITLE
HEEDLS-649 - make delegate progress field names consistent

### PR DESCRIPTION
**Pull request by DanBloxham-sw**
### JIRA link
https://softwiretech.atlassian.net/browse/HEEDLS-649

### Description
Made changes to the Course Delegates cards, the View Delegate course cards, and the Delegate Progress page to make the names and display of various fields consistent. I'll comment on 567's PR to make sure the Detailed Progress page also has consistent names where appropriate.

### Screenshots
Delegate progress page:
Delegate Progress

View delegate course card:
View Delegate cropped

Course delegate card:
Course delegate card

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [x] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

**Review by AlexJacksonDS — CHANGES_REQUESTED** [review]
Just one comment on a potential missed string

**Comment by GeorgeBlackwell** [comment]
Looks good! Just the one string Alex raised as a protentional change

**Review by GeorgeBlackwell — APPROVED** [review]
Looks good! Just the one string Alex raised as a protentional change

File: DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompletionDate.cshtml [hunk 1]
```diff
@@ -22,7 +20,9 @@
     cancelLinkController = "ViewDelegate";
   }
   var exampleDate = DateTime.Today;
-  var hintTextLines = new List<string> { $"For example, {exampleDate.Day} {exampleDate.Month} {exampleDate.Year}. Leave the boxes blank to clear the completion date" };
+  var hintTextLines = new List<string> {
+    $"For example, {exampleDate.Day} {exampleDate.Month} {exampleDate.Year}. Leave the boxes blank to clear the completion date",
```
[the remark this answers is not in the document]
Comment (DanBloxham-sw) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompletionDate.cshtml]: Good spot, changed it.

**Comment by DanBloxham-sw** [comment]
A question remains about the diagnostic score display when null https://github.com/TechnologyEnhancedLearning/DLSV2/pull/942#discussion_r812636940

**Pull request merged**